src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+);
```

Comment:
- [ ] When I start the application, the screen is blank. After creating the store, you need to connect to the store using the Provider component from redux. Kindly apply this change. 👍 😎